optaplanner-examples/pom.xml

```diff
@@ -175,6 +175,32 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>javax.json</groupId>
+      <artifactId>javax.json-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.glassfish</groupId>
+      <artifactId>javax.json</artifactId>
+    </dependency>
```

Comment:
@jhrcek could you please check the build failure?
enforce-direct-dependencies on optaplanner-examples is failing after I added these two dependencies although the pom tree doesn't show any transitive dependencies.